### ISSUE TITLE
Unable to install via bower

### ISSUE THREAD

**Issue by pallaviMN** [shprink/ionic-native-transitions#107]
Hi Team,  

I am trying to install native transition via bower , but getting below error 

bower                     invalid-meta The "name" is recommended to be lowercase, can contain digits, dots, dashes
bower ionic-native-transitions#*       not-cached git://github.com/shprink/ionic-native-transitions.git#*
bower ionic-native-transitions#*          resolve git://github.com/shprink/ionic-native-transitions.git#*
bower ionic-datepicker#0.9.0               cached https://github.com/rajeshwarpatlolla/ionic-datepicker.git#0.9.0
bower ionic-datepicker#0.9.0             validate 0.9.0 against https://github.com/rajeshwarpatlolla/ionic-datepicker.git#0.9.0
bower ionic-native-transitions#*          ECMDERR Failed to execute "git ls-remote --tags --heads git://github.com/shprink/ionic-native-transitions.git", exit code of #128 fatal: repository 'https://github.com/shprink/ionic-native-transitionsgithub.com/shprink/ionic-native-transitions.git/' not found

Additional error details:
fatal: repository 'https://github.com/shprink/ionic-native-transitionsgithub.com/shprink/ionic-native-transitions.git/' not found

**Comment by shprink** [comment]
yes, please use npm or try to make it work on bower (PR welcome)! thanks

**Comment by pallaviMN** [comment]
@shprink , i tried installing even via npm , i got below error


C:\Users\pallavi.mn\Desktop\InsightApp8thFeb>npm install ionic-native-transitions --save
npm WARN package.json insightapp@1.0.0 No repository field.
npm WARN package.json insightapp@1.0.0 No README data
npm WARN package.json insightapp@1.0.0 No license field.
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "ionic-native-transitions" "--save"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: ionic-native-transitions@'*'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["1.0.0-alpha1","1.0.0-alpha2","1.0.0-alpha4","1.0.0-alpha5","1.0.0-beta1","1.0.0-beta2","1.0.0-beta3","1.0.0-beta4","1.0.0-beta5","1.0.0-beta6","1.0.0-rc1","1.0.0-rc2","1.0.0-rc3","1.0.0-rc4","1.0.0-rc5","1.0.0-rc6","1.0.0-rc7","1.0.0-rc8","1.0.0-rc9","1.0.0-rc10","1.0.0-rc11"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'insightapp'
npm ERR! notarget

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\pallavi.mn\Desktop\InsightApp8thFeb\npm-debug.log

**Comment by shprink** [comment]
Weird. What version of node and npm are you running?

**Comment by pallaviMN** [comment]
@shprink  node.js - v0.12.7 & npm 2.11.3

**Comment by shprink** [comment]
Try with npm 3.x, as for you node version you should probably update to 5 or 6.

**Comment by pallaviMN** [comment]
Can you please tell me the command to update it?

**Comment by pallaviMN** [comment]
or just download new node version?

**Comment by shprink** [comment]
depends on your OS
